mitosheet/mitosheet/tests/test_transpile.py

```diff
@@ -597,28 +598,28 @@ def test_transpile_fully_parameterized_function_string(tmp_path):
     assert mito.mito_backend.fully_parameterized_function == f"""from mitosheet.public.v3 import *
 import pandas as pd
 
-def function(import_dataframe_0, file_name_import_csv_0, file_name_import_excel_0, file_name_export_csv_0, file_name_export_excel_0):
+def function({expected_in_function}, file_name_import_csv_0, file_name_import_excel_0, file_name_export_csv_0, file_name_export_excel_0):
     txt = pd.read_csv(file_name_import_csv_0)
     
     sheet_df_dictonary = pd.read_excel(file_name_import_excel_0, engine='openpyxl', sheet_name=[
         'Sheet1'
     ], skiprows=0)
     Sheet1 = sheet_df_dictonary['Sheet1']
     
-    import_dataframe_0.to_csv(file_name_export_csv_0, index=False)
+    {expected_in_function}.to_csv(file_name_export_csv_0, index=False)
     
     with pd.ExcelWriter(file_name_export_excel_0, engine="openpyxl") as writer:
         txt.to_excel(writer, sheet_name="txt", index=False)
     
-    return import_dataframe_0, txt, Sheet1
+    return {expected_in_function}, txt, Sheet1
 """
     assert mito.mito_backend.param_metadata == [
         {
-            'initial_value': 'test_df_name',
+            'initial_value': 'test_df_name' if df_names else '',
```

Comment:
Do we want to make it so we get the dataframe name in this case, rather than just returning an empty string? Likely, we'd want to handle this in the the OG get parametrizable params functions...